### ISSUE TITLE
When sorting by a field, make the value of the field for each app visible in the app list

### ISSUE THREAD

**Issue by surlyhacker** [MuntashirAkon/AppManager#1148]
### Please check before submitting an issue

- [X] I am using the latest version of App Manager
- [X] I have searched the issues and haven't found anything relevant
- [X] I have read the docs

### Describe a description of the new feature

The app list (main page) allows many types of sorting, and there are several fields that can be selected as a sort criterion that are not listed in the [information card that appears for each app in the main page list](https://muntashirakon.github.io/AppManager/en/#sec:main-page)

I think that any time a list is sorted, the values shown for each item in the list should contain the value by which it is sorted.

In other words, if I sort by "Installation date", then the installation date of each app should be shown in the list.
If I sort by "Last used", then the last used date of each app should be shown in the list.

**Comment by MuntashirAkon** [comment]
Continuing a similar discussion from the Matrix room, we have a different view on sorting and filtering because there are at least 43 indices (excluding the per filter option that may be required depending on the option) on which sorting and filtering can be applied, and it's exhausting every time someone wants their preferred customisation to be present in the list options. We're going to introduce a rule-based filtering where user can customise those 43 indices based on their needs along with some presets/templates with the most common or frequently used filters.

**Comment by MuntashirAkon** [comment]
Due to the reasons stated above, a similar feature will be present in a dedicated “Finder” page.
